src/main/java/org/openjdk/backports/jira/InterestTags.java

```diff
@@ -32,40 +32,47 @@
 public class InterestTags {
 
     public static String shortTags(Collection<String> labels) {
-        List<String> comps = new ArrayList<>();
+        List<String> tags = new ArrayList<>();
         for (String label : labels) {
-            if (!label.endsWith("-interest")) continue;
-            comps.add(label.substring(0, label.indexOf("-interest")));
+            String tag = shortTagFor(label);
+            if (!tag.isEmpty()) {
+                tags.add(tag);
+            }
         }
 
-        if (comps.isEmpty()) return "";
+        if (tags.isEmpty()) return "";
 
-        Collections.sort(comps);
+        Collections.sort(tags);
 
         StringBuilder sb = new StringBuilder();
         sb.append("[");
-        for (String comp : comps) {
-            sb.append(shortTagFor(comp));
+        for (String tag : tags) {
+            sb.append(tag);
         }
         sb.append("]");
         return sb.toString();
     }
 
-    private static String shortTagFor(String company) {
-        switch (company) {
-            case "redhat":
+    private static String shortTagFor(String label) {
+        switch(label) {
```

Comment:
Retain the space here: `switch (label)`

Reply:
Thanks - I've addressed and forced-pushed an update.